### DISCUSSION TITLE
Pushnotifications using Azure Notification Hub

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#12961] [Q&A]
Hello.

Does anyone know or have a tutorial like this for using Azure Notification Hub for .net maui?
[https://learn.microsoft.com/en-us/azure/developer/mobile-apps/notification-hubs-backend-service-xamarin-forms](https://learn.microsoft.com/en-us/azure/developer/mobile-apps/notification-hubs-backend-service-xamarin-forms)

Thanks!

**Comment by FM1973** [comment]
At least for Android I found a repository which helps;
[https://github.com/ms-us-rcg-app-innovation/notifications-and-messaging](https://github.com/ms-us-rcg-app-innovation/notifications-and-messaging)

**Reply by Denny966** [reply]
The link is broken, do you have an example?

**Reply by FM1973** [reply]
Have a look at [https://github.com/Azure/azure-notificationhubs-xamarin/issues/125#issuecomment-1411732544](https://github.com/Azure/azure-notificationhubs-xamarin/issues/125#issuecomment-1411732544)

**Comment by corne-ac** [comment]
I know this is a bit late, but for anyone passing buy, this helped me implement notifications for android, the solution worked for me, there's some issues here and there, but nothing that cant easily be resolved: https://puresourcecode.com/dotnet/maui/maui-push-notifications-using-azure-notification-hub-for-android/
Start off by following the normal setup for windows (do only the azure notifications hub there) then switch back to the android guide and follow it from there. I have yet to do the IOS, but im sure it will help.
Finally, dont hesitate to sponsor the guy on github, its clear a lot of work went into figuring out these implementations.

**Reply by developer9969** [reply]
do you have a working repo that we can look at? Also the android instructions seems to be releated to the now deprecated FCM vs FCMV1 thanks
